plotly/figure_factory/_county_choropleth.py

```diff
@@ -557,7 +557,7 @@ def create_choropleth(fips, values, scope=['usa'], binning_endpoints=None,
     """
     # ensure optional modules imported
     if not gp or not shapefile or not shapely:
-        raise ImportError("geopandas, shapefile and shapely must be "
+        raise ImportError("geopandas, pyshp and shapely must be "
                           "installed for this figure factory")
```

Comment:
Should we add the pip statements here? For example, "Try pip install geopandas and pip install pyshp in your terminal."

Reply:
good idea. we can add it to the docs as well to remind in two places

Reply:
> Just to confirm

confirmed